modules/builds-create-imagestreamtag.adoc

```diff
@@ -3,29 +3,26 @@
 //* builds/running-entitled-builds.adoc
 
 [id="builds-create-imagestreamtag_{context}"]
-= Creating an image stream tag to the Red Hat Universal Base Image
```

Comment:
I changed all of the general references for image stream with term updates before shutdown (in the build book; images is still in progress) and treated image stream tag/`imagestreamtag` in the same manner. I wanted to give you some background, but please change it as necessary. 

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
@bmcelvee I've updated the file based on my understanding of your comment and the guidelines. PTAL